src/Craft.php

```diff
@@ -173,7 +173,9 @@ public static function cookieConfig(array $config = [], ?Request $request = null
                     $request = static::$app->getRequest();
                 }
 
-                $generalConfig->useSecureCookies = $request->getIsSecureConnection();
+                if (!$request->isConsoleRequest) {
+                    $generalConfig->useSecureCookies = $request->getIsSecureConnection();
+                }
             }
```

Comment:
Required check since the this method will now be called during bootstrap. Alternatively we could only set the DI definition for `Cookie` on web requests, but I preferred just adding this check.